exercises/removeSpecialCharacters/rmSpecialChars_algo.cpp

```diff
@@ -0,0 +1,31 @@
+#include <algorithm>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <vector>
+
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include "../../doctest.h"
+
+std::string rmSpecialCharacters(std::string&& result) {
```

Comment:
in questo caso si puo' passare by value; se l'argomento passato e' un rvalue, viene usato cmq il move constructor; se e' un lvalue, comunque bisogna fare una copia, quindi tanto vale farla nella chiamata